app/etc/di.xml

```diff
@@ -1800,4 +1790,26 @@
     </type>
     <preference for="Magento\Framework\GraphQl\Query\ErrorHandlerInterface" type="Magento\Framework\GraphQl\Query\ErrorHandler"/>
     <preference for="Magento\Framework\Filter\VariableResolverInterface" type="Magento\Framework\Filter\VariableResolver\StrategyResolver"/>
+    <!-- -->
+    <type name="Magento\Framework\DB\Logger\FileFactory">
+        <arguments>
+            <argument xsi:type="string" name="instanceName">Magento\Framework\DB\Logger\Monolog</argument>
+        </arguments>
+    </type>
+    <virtualType name="Magento\Framework\Logger\Handler\System1" type="\Magento\Framework\Logger\Handler\StreamHandler">
+        <arguments>
+            <argument name="formatter" xsi:type="object">Magento\Framework\Logger\Formatter\JsonFormatter</argument>
+            <argument name="stream" xsi:type="string">php://stdout</argument>
+            <argument name="level" xsi:type="const">Monolog\Logger::DEBUG</argument>
```

Comment:
As I understand, this means all messages of level "debug" and higher will be logged.
This makes sense for dev environment, but is usually configured differently in production. It would be good to have an ability to configure level of logging from env variable (or env.php).